src/managing-workflow/tuning-component-settings.md

```diff
@@ -50,9 +50,10 @@ limits_memory = "50Mi"
 
 The following environment variables are tunable for the [Builder][] component:
 
-Setting | Description
-------- | ---------------------------------
-DEBUG   | Enable debug log output (default: false)
+Setting                     | Description
+--------------------------- | ---------------------------------
+DEBUG                       | Enable debug log output (default: false)
+BUILDER_POD_NODE_SELECTOR   | A node selector setting for builder job. As builder job sometimes consume a lot of node resources. In case you want a builder job to run in specific node only, so it won't effect the critical node. for example `pool:testing,disk:magnetic`
```

Comment:
Minor grammatical suggestion -- perhaps change to something like:

>As it may sometimes consume a lot of node resources, one may want a given builder job to run in a specific node only, so it won't affect critical nodes.

Otherwise, thank you for adding!